build.sh

```diff
@@ -101,12 +101,15 @@ fi
 
 pushd "$BUILD_DIR"
 
+# avoid the command failed and exits
```

Comment:
Which kinds of failures can happen here? Previously there was no || true necessary here, right?

Reply:
the script will exit if error happen as `set -e` at the begin.

So the first time, `make clean` in an empty directory which leads to `makefile` not found error.

`|| true` will redirect the error and keep the script to continue. 

I use some directories existence to justify whether the lib is built already. once `make clean` is called, the directories will be removed.

However, if I call `cmake` command first, then cmake will see the directories exist. And it will not build the libs any more. Then I can `make clean`, which leads to the directories removed. So the error happens.

Reply:
Thanks for the explanation!